node-bootstrapper/parser/helper.go

```diff
@@ -673,12 +673,12 @@ func getInitAKSCustomCloudFilepath() string {
 }
 
 func getGPUNeedsFabricManager(vmSize string) bool {
-	return common.GPUNeedsFabricManager(vmSize)
+	return agent.GPUNeedsFabricManager(vmSize)
 }
 
 func getEnableNvidia(config *nbcontractv1.Configuration) bool {
 	if config.GpuConfig != nil && config.GpuConfig.EnableNvidia != nil {
 		return *config.GpuConfig.EnableNvidia
 	}
-	return common.IsNvidiaEnabledSKU(config.GetVmSize())
+	return false
```

Comment:
This function is gone. Is it ok to do that?

Reply:
Yes, this is fine.